core/chains/evm/gas/rollups/l1_gas_price_oracle.go

```diff
@@ -65,6 +65,13 @@ const (
 	// `function l1BaseFee() external view returns (uint256);`
 	KromaGasOracle_l1BaseFee = "519b4bd3"
 
+	// GasOracleAddress is the address of the precompiled contract that exists on scroll chain.
+	// This is the case for Scroll.
+	ScrollGasOracleAddress = "0x5300000000000000000000000000000000000002"
+	// GasOracle_l1BaseFee is the a hex encoded call to:
+	// `function l1BaseFee() external view returns (uint256);`
+	ScrollGasOracle_l1BaseFee = "519b4bd3"
+
 	// Interval at which to poll for L1BaseFee. A good starting point is the L1 block time.
 	PollPeriod = 12 * time.Second
```

Comment:
Actually found one thing. For the gas estimator to initialize the L1 oracle for Scroll, we may need to add it to ths supportedChainTypes list below. I couldn't add the comment to the actual line. This list would be used [here](https://github.com/smartcontractkit/chainlink/blob/c2115d32801930fe9448b8b0ed036c2356c94776/core/chains/evm/gas/rollups/l1_gas_price_oracle.go#L72).

Reply:
added! thank you for catching this